src/fides/api/ops/service/connectors/consent_email_connector.py

```diff
@@ -207,6 +237,13 @@ def batch_email_send(self, privacy_requests: Query) -> None:
             if filtered_user_identities and (
                 consent_preference_schemas or filtered_privacy_preference_records
             ):
+                cache_initial_status_and_identities_for_consent_reporting(
+                    privacy_request=privacy_request,
+                    connection_config=self.configuration,
+                    relevant_preferences=filtered_privacy_preference_records,
+                    relevant_user_identities=filtered_user_identities,
+                )
```

Comment:
This other scenario handles when potentially only a subset of privacy preferences on a privacy request should be propagated.  So some are potentially getting pending statuses and some are given skipped statuses